resources/js/Layouts/PraktikanAuthenticatedLayout.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import { Link } from "@inertiajs/react";
+import PraktikanNav from "../Components/ComponentsPraktikans/PraktikanNav";
+import PraktikanAnnouncement from "../Components/ComponentsPraktikans/PraktikanAnno";
+
+export default function PraktikanAuthenticated({ children }) {
```

Comment:
This should be okay for now, but just note that this should be refactored later when you guys want to connect with the backend side, because the correct way is to separate dashboard for non authenticated account and dashboard for authenticated account instead of putting the layout for both authenticated and non authenticated on the same dashboard with a check